src/main/java/seedu/address/model/person/github/repo/Repo.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.model.person.github.repo;
+
+/**
+ * Class for each individual repository for a specified GitHub.
+ */
+public class Repo {
+    private String repoName;
+    private String language = "";
+    private int numberForks = 0;
+    private String license = "";
+    private String lastUpdated = "";
```

Comment:
Should we follow other `Attribute` classes? To make it immutable.

Reply:
I didn't think so as the fields are being populated by responses from the GitHub API.